src/makerspace/templates/makerspace/statistics.html

```diff
@@ -0,0 +1,68 @@
+{% extends 'web/base.html' %}
+{% load static %}
+{% load i18n %}
+
+
+{% block title %}{% translate "Statistics" %}{% endblock title %}
+
+{% block extra_head %}
+    <script defer src="{% static 'makerspace/js/statistics_charts.js' %}"></script>
+{% endblock extra_head %}
+
+{% block body%}
+<div class="ui container">
+    <h1 class="ui text-centered make-col-blue header"> {% translate "Statistics" %}</h1>
+    <div class="ui column cards">
+    <div class="card " style="height: 50vh; width: 65vh">
+            <div class="content">
+            <div class="header">{% translate "How long each 3D-printer* has been reserved in total:" %} </div>
+            {{ span_of_printer_reservations|json_script:"span_of_printer_reservations" }}
+            <canvas id="span_printer_reservations" width="800" height="550">
+            </canvas>
+            </div>
+        </div>
+
+    <div class="card" style="height: 50vh; width: 65vh">
+            <div class="content">
+            <div class="header"> {% translate "Number of times each 3D-printer* has been reserved:" %} </div>
```

Comment:
I also find "Antall ganger hver 3D-printer har blitt reservert" a bit complicated. Maybe we could write "Antall reservasjoner pr. 3D-printer" (Norwegian) / "Number of reservations pr. 3D-printer" (English). Same goes for sewing machines.